package.json

```diff
@@ -6,7 +6,6 @@
     "@azure/storage-blob": "^12.11.0",
     "@google-cloud/storage": "^6.9.3",
     "@techteamer/ocsp": "1.0.0",
-    "agent-base": "^6.0.2",
```

Comment:
I removed this and several other packages that are no longer used now that I've updated `https_proxy_ocsp_agent.js`.